public/locales/pseudo-LOCALE/grafana.json

```diff
@@ -369,7 +369,6 @@
       "title-label": "Ŧįŧľę"
     },
     "json-editor": {
-      "apply-button": "Åppľy čĥäŉģęş",
```

Comment:
LOL! What is this?!?!?!

Reply:
Translation extraction stuff, the default pseudo-locale is funky. CI was complaining about it but seems to be good now 🤔